Composer/ScriptHandler.php

```diff
@@ -217,7 +217,7 @@ public static function installRequirementsFile(Event $event)
         // as the file must be removed for production use
         if ($fs->exists($webDir.'/config.php')) {
             if (!$newDirectoryStructure) {
-                $fs->copy(__DIR__.'/../Resources/skeleton/web/config.php', $webDir.'/config.php', true);
+                $fs->dumpFile($webDir.'/config.php', str_replace('/../app/SymfonyRequirements.php', '/'.$fs->makePathRelative($appDir, $webDir).'SymfonyRequirements.php', file_get_contents(__DIR__.'/../Resources/skeleton/web/config.php')));
```

Comment:
As this line is almost identical to the one in else, can you set a temp variable with $appDir or $varDir and move the command after the condition?